quactography/solver/multiprocess_solver.py

```diff
@@ -0,0 +1,26 @@
+import multiprocessing
+import itertools
+
+from quactography.solver.qaoa_solver import _find_longest_path
+
+alpha_min_costs = []
+
+
+def multiprocess_qaoa_solver(hamiltonians, reps, nbr_processes, output_file):
+    pool = multiprocessing.Pool(nbr_processes)
+
+    results = pool.map(
+        _find_longest_path,
+        zip(
+            hamiltonians,
+            itertools.repeat(reps),
+            itertools.repeat(output_file),
+        ),
+    )
+    pool.close()
+    pool.join()
+
+    print(
+        "------------------------MULTIPROCESS SOLVER FINISHED-------------------------------"
+    )
+    return results
```

Comment:
Remove the return if you never use it. I understand that you save the npz inside `_find_longest_path` and don't use `results` after. A "cleaner" way would be to save the output in the script rather than in the function, but this is not mandatory.